DFC.ServiceTaxonomy.CustomEditor/Views/ContentsButtonActions_SummaryAdmin.cshtml

```diff
@@ -25,6 +25,12 @@
     var hasPreviewPermission = await AuthorizationService.AuthorizeAsync(User, CommonPermissions.PreviewContent, contentItem);
     var previewContentTypes = contentItem.ContentType == ContentTypes.Page || contentItem.ContentType == ContentTypes.JobProfile || contentItem.ContentType == ContentTypes.Footer || contentItem.ContentType == ContentTypes.Header;
 
+    // We do not display the Clone button if the content type is a header or footer.
```

Comment:
same as above comment  call the authorisation service if not in any of the content type.

Reply:
Updated.